Added AutomationViewBuilderFactory changes to upgrade.md (#59)
sulu/SuluAutomationBundle: UPGRADE.md

The bundle was so refractored to be compatible with [Sulu 2.0](https://github.com/sulu/sulu/blob/2.x/UPGRADE.md#200).

### AutomationViewBuilder refactored to AutomationViewBuilderFactory

> For more consistency with the core bundle, the `AutomationViewBuilder` has been refactored to the `AutomationViewBuilderFactory`.

Reviewer: ```suggestion
The `AutomationViewBuilder` has been refactored and renamed to `AutomationViewBuilderFactory` to be consistent with the Sulu core.
```

In order to add the automation tab to a custom view. You now have to inject the `AutomationViewBuilderFactory` into your
custom admin class.

> #### Custom-Admin class
> ```diff

Reviewer 2: some markdown readers requires empty lines before new statements to render correctly.

```suggestion
#### Custom-Admin class

```diff
```
